### PR TITLE
Issue #3943: Add examples for xdoc for ArrayTrailingComma

### PR DESCRIPTION
**Pull request by Pedro12909**
Issue: https://github.com/checkstyle/checkstyle/issues/3943

### CODE REVIEW

File: src/xdocs/config_coding.xml [hunk 1]
```diff
@@ -50,12 +50,31 @@ return new int[] { 0 };
...
+          </source>
+          <p>
+            Which results in the following violations:
+          </p>
+          <source>
+            int[] numbers = {1, 2, 3};        //no violation
+            boolean[] bools = {
+            true,
+            true,
+            false        //violation
```
Comment (sabaka) [inline src/xdocs/config_coding.xml]: Violation will be displayed on the next line

File: src/xdocs/config_coding.xml [hunk 2]
```diff
@@ -50,12 +50,31 @@ return new int[] { 0 };
...
+            };
+
+            String[][] text = {{},{},};        //no violation
+
+            double[][] decimals = {
+            {0.5, 2.3, 1.1,},        //no violation
+            {1.7, 1.9, 0.6},
+            {0.8, 7.4, 6.5}        //violation
```
Comment (sabaka) [inline src/xdocs/config_coding.xml]: Violation will be displayed on the next line

File: src/xdocs/config_coding.xml (hunk 1 above)
Comment (Pedro12909) [inline src/xdocs/config_coding.xml]: Done

File: src/xdocs/config_coding.xml (hunk 2 above)
Comment (Pedro12909) [inline src/xdocs/config_coding.xml]: Done

**Pull request merged**